### ISSUE TITLE
Add a how-to on running the project

### ISSUE THREAD

**Issue by t1-tracey** [ncss/projects-2017-3#14]
The how-to will be for anyone to look at, like from other groups, or for example I'll look at it if I've forgotten how to do it at any point in future :)

**Comment by t1-tracey** [comment]
Are there conventions for this sort of thing? For example, is it usually done in the README.md, or is there another place where you could put it?

**Comment by t1-tracey** [comment]
I'll just quickly note down things for the how-to.

- In GitHub Desktop
- Clone repo (Screenshot) *only if first time*
- Open Terminal/Git Shell (Screenshot again with note this is Mac version)
- basic command is `python file.py` to run a file named `file.py`
- only if first time running, need to create the db. `python db_create.py` (Screenshot with note that `python3` is for Mac)
- each time you want to run the project you need the server, run `python server.py` (Screenshot)
- open a browser, type in `localhost:8888`
- (insert pic of site)

- Can view in Atom by going to Repository>Open in Atom (Insert screenshot with note this is Mac version)

**Comment by lyneca** [comment]
Yes, this is normally done in the `README.md`. [Here's an example](https://github.com/lyneca/facebook-messenger-tool/blob/master/README.md) (and a shameless self-promotion 😛).

**Comment by t1-tracey** [comment]
I'm gonna do a little draft here and put screenshots ^_^ this will be meant for totally starting at the start, like if I've forgotten everything or am doing it on another device

-this is free to edit at any point. I'm editing it too

# Draft

- this is quickly for in GitHub Desktop
- log in screenshot
- Clone repo (Screenshot) *only if first time*
- Open Terminal/Git Shell (Screenshot again)
- basic command is `python file.py` to run a file named `file.py`
- only if first time running, need to create the db. `python db_create.py` (Screenshot with note that `python3` is for Mac for example)
- each time you want to run the project you need the server, run `python server.py` (Screenshot)
- open a browser, type in `localhost:8888`
- (insert pic of site)

- and can view in Atom by going to Repository>Open in Atom (Insert screenshot with note this is Mac version)

**Comment by t1-tracey** [comment]
@lyneca 👍

**Comment by lyneca** [comment]
I've done some stuff in pull request #15

**Comment by t1-tracey** [comment]
That's awesome 👍 we just need to create the database the first time running the project though — for the dummy data I think, so I think we can just mention that's for the first time, and each time afterwards you just need `python server.py`

**Comment by t1-tracey** [comment]
I've heard of `bash` but not sure what it does, what does it do? 😊

**Comment by lyneca** [comment]
`bash` is a linux shell - like a language for terminals

**Comment by t1-tracey** [comment]
oh okay, thank you 😊 I'm not quite sure still but it's okay 👍

**Comment by t1-tracey** [comment]
Okay I think I'm done with the draft for now ☺️ I can just put it as a more in-depth/newbie-friendly version somewhere, don't know where though. I don't know how the images will load if I put it in a file?

**Comment by lyneca** [comment]
Images can be put in a readme.
```markdown
![Alt text](http://full/path/to/img.jpg "Optional title")
```
Put the images on [imgur](https://imgur.com) or something and put the full image URL in that bit of code, such as this:
```markdown
![picture of a cat](https://i.imgur.com/F60i2Ga.jpg "Cat Photo")
```
![picture of a cat](https://i.imgur.com/F60i2Ga.jpg "Cat Photo")

Get the full image path by right clicking on the image and hitting "copy image address".

**Comment by lyneca** [comment]
Wikis are used for making user-created documentation for a project. Probably not what we're looking for in this case.

**Comment by t1-tracey** [comment]
Oooh okay thank you for the how-to on images in Markdown 😄 oh okay, I see for wikis now, yep probably not for this.

hmm I was thinking about either updating `README.md` with that extra info, or I could add it to `/documentation`? I don't know

**Comment by lyneca** [comment]
I think put it in `README.md`; it's the document that shows up when you first look at the project.

**Comment by t1-tracey** [comment]
cool thanks Luke :D

**Comment by t1-tracey** [comment]
Once this is merged https://github.com/ncss/projects-2017-3/pull/17 I'll close this 😃

**Comment by t1-tracey** [comment]
— as I mentioned in last commit
> The images are a bit big and some of the spacing is off

so always free to edit :) the main things are there so I'm just gonna leave it for now

**Comment by t1-tracey** [comment]
just updating the how-to with another image instead, uploading it here so I can use the link hehe..

<img width="438" alt="screen shot 2017-01-22 at 2 17 06 pm" src="https://cloud.githubusercontent.com/assets/22441348/22179660/b2671018-e0ae-11e6-8df9-b70c499bd306.png">

**Comment by t1-tracey** [comment]
Now it's merged into `master` thanks to @juliawong 😄

**Comment by tracey-le** [comment]
basic how-to is there so I closed this

ideas for improvement (can re-open issue or make another one):

- [x] upload smaller images and re-link the image links in README.md (or experiment and see if html `<img>` elements work?)
- [x] fix some of the spacing (try `<br>`?)

**Comment by tracey-le** [comment]
yay I've done those little changes now, with the resized images and the spacing :) :D
